python/Makefile.am

```diff
@@ -0,0 +1,11 @@
+SWIG_SOURCES = analysis.i
+
+# TODO: Check libdivvun to remove call to all
+all: analysis_wrap.cpp
```

Comment:
```suggestion
all: analysis_wrapper.cpp
```

Reply:
the output of swig for `foo.i` is `foo_wrap.cxx` http://swig.org/Doc4.0/SWIGDocumentation.html#SWIG_output, which can be modified by `-o` flag. Should I rename it to `analysis_wrapper`?

Reply:
Ah, in that case, leave it as is for now.